frontend/src/modules/Traces/TraceGanttChart/TraceGanttChart.tsx

```diff
@@ -229,24 +229,26 @@ const TraceGanttChart = ({
 	};
 
 	const handleResetFocus = () => {
-		let rows = document.querySelectorAll("#collapsable table tbody tr");
+		const rows = document.querySelectorAll('#collapsable table tbody tr');
 		Array.from(rows).map((row) => {
-			row.classList.remove("hide");
+			row.classList.remove('hide');
 		});
 
 		resetZoom(true);
 	};
 
-	const handleScroll = (id) => {
-		let rows = document.querySelectorAll("#collapsable table tbody tr");
-		const table = document.querySelectorAll("#collapsable table");
-		Array.from(rows).map((row) => {
-			let attribKey = row.getAttribute("data-row-key");
-			if (id === attribKey) {
-				let scrollValue = row.offsetTop;
-				table[1].scrollTop = scrollValue;
-			}
-		});
+	const handleScroll = (id: string): void => {
+		if (!isEmpty(id)) {
+			const rows = document.querySelectorAll<HTMLElement>('#collapsable table tbody tr');
+			const table = document.querySelectorAll<HTMLElement>('#collapsable .ant-table-body');
+			Array.from(rows).map((row) => {
```

Comment:
We should use `forEach` instead of `map`

Reply:
Any specific reason apart from map returns an array and we are not consuming it?

Reply:
Yes we are not consuming the value, so we can use forEach in this case